package-scripts.js

```diff
@@ -71,10 +71,17 @@ module.exports = {
       ci: 'prettier --list-different "./**/*.{js,jsx,json,ts,tsx}"',
       default: "yarn nps format.fix"
     },
+    typecheck: {
+      default: npsUtils.series.nps("typecheck.core"),
+      base: "tsc --noEmit",
+      demo: "tsc -p ./demo/tsconfig.json --noEmit",
+      core: "lerna exec --scope victory-core -- nps typecheck.base"
+    },
     check: {
       ci: npsUtils.series.nps(
         "format.ci",
         "lint",
+        "typecheck",
```

Comment:
Separately take a look at everything with `compile-ts` -- I think we want to integrate/reove that?

Reply:
Oh, I missed that script, looking into it now.  Seems like it's the same as the `typecheck.base` script I just created.

From what I can tell, all our previous TypeScript configuration was being used for the `demo` folder, right?  I will try to make sure I test compiling that folder.

Reply:
I think we both generate typedefs like: https://unpkg.com/browse/victory@36.4.1/es/index.d.ts and we have the TS demo